BUILDING.md

```diff
@@ -491,6 +492,18 @@ $ gdb /opt/node-debug/node core.node.8.1535359906
 $ backtrace
 ```
 
+#### Building an ASAN build
+
+[ASAN](https://github.com/google/sanitizers) build currently only support linux.
+If you want to check it on windows or macOS, you can try
+[docker](https://www.docker.com/products/docker-desktop)
+ (using Image like `gengjiawen/node-build:2020-02-14`)
```

Comment:
```suggestion
[Docker](https://www.docker.com/products/docker-desktop) may be
used on either Windows or macOS using an image such as
`gengjiawen/node-build:2020-02-14`.
```

Reply:
I think this paragraph is misleading though, since this is still a Linux build and sanitizer is only covering Linux code paths. Besides, you can also use the Docker image on Linux. Maybe drop the "on either Windows or macOS"?

Reply:
I rephrased it, please take a look.